databricks/koalas/base.py

```diff
@@ -178,7 +178,7 @@ def spark_column(self) -> Column:
     # arithmetic operators
     __neg__ = column_op(Column.__neg__)
 
-    def __add__(self, other):
+    def __add__(self, other) -> "ks.Series":
```

Comment:
`Union["ks.Series", "ks.Index"]`? and ditto for the following binary operations?

Reply:
Makes sense! Modified.